index.bs

```diff
@@ -2000,7 +2025,14 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
                         <dl class="switch">
 
                             :   is set to {{UserVerificationRequirement/required}}
-                            ::  Let |userVerification| be [TRUE].
+                            ::  If <code>|options|.{{CredentialCreationOptions/mediation}}</code> is set to
+                                <dl class="switch">
+                                    :   {{CredentialMediationRequirement/conditional}}
+                                    ::  throw a {{NotAllowedError}} {{DOMException}} unless it can be explicitly collected during the ceremony.
+
+                                    :   empty or another value
+                                    ::  Let |userVerification| be [TRUE].
+                                </dl>
```

Comment:
Also, is `NotAllowedError` the right behaviour here? The closest analogue currently in the spec is the "error code equivalent to `ConstraintError`" returned by the authenticator when UV is required but cannot be satisfied by that authenticator. Returning an error at all also differs from `conditional` mediation in `get()`, which simply hangs forever on most errors. Should `conditional` create do that too?

Reply:
The idea with conditional registration is that you invoke it right after logging in and get a result quickly. If the user didn't consent to this type of create, then they will quickly get an error, otherwise a credential. 

I think `ConstraintError` does sound okay in this case. I've made the change.